core/dive.c

```diff
@@ -714,18 +725,18 @@ void copy_cylinders(struct dive *s, struct dive *d, bool used_only)
 	}
 	for (i = j = 0; i < MAX_CYLINDERS; i++) {
 		if (!used_only || is_cylinder_used(s, i) || s->cylinder[i].cylinder_use == NOT_USED) {
-			d->cylinder[j].type = s->cylinder[i].type;
-			d->cylinder[j].type.description = copy_string(s->cylinder[i].type.description);
-			d->cylinder[j].gasmix = s->cylinder[i].gasmix;
-			d->cylinder[j].depth = s->cylinder[i].depth;
-			d->cylinder[j].cylinder_use = s->cylinder[i].cylinder_use;
-			d->cylinder[j].manually_added = true;
-
-			// Restore the start and end pressures from original cylinder
-			d->cylinder[i].start.mbar = t[i].start.mbar;
-			d->cylinder[i].end.mbar = t[i].end.mbar;
-			d->cylinder[i].sample_start.mbar = t[i].sample_start.mbar;
-			d->cylinder[i].sample_end.mbar = t[i].sample_end.mbar;
+		d->cylinder[j].type = s->cylinder[i].type;
+		d->cylinder[j].type.description = copy_string(s->cylinder[i].type.description);
+		d->cylinder[j].gasmix = s->cylinder[i].gasmix;
+		d->cylinder[j].depth = s->cylinder[i].depth;
+		d->cylinder[j].cylinder_use = s->cylinder[i].cylinder_use;
+		d->cylinder[j].manually_added = true;
+
+		// Restore the start and end pressures from original cylinder
+		d->cylinder[i].start.mbar = t[i].start.mbar;
+		d->cylinder[i].end.mbar = t[i].end.mbar;
+		d->cylinder[i].sample_start.mbar = t[i].sample_start.mbar;
+		d->cylinder[i].sample_end.mbar = t[i].sample_end.mbar;
```

Comment:
that indentation change looks bogus to me

Reply:
Yeah, that's completely bogus